### ISSUE TITLE
Can't use single buffering with EGL as context creation api

### ISSUE THREAD

**Issue by AaxAisa** [glfw/glfw#1843]
Hello!
I'm working on ARM and there's very few options when it comes to how I can go around creating graphics there. Namely, I have to use EGL as context creation API and Opengl ES as the context.

Now, I noticed some odd performance dips in my recent project, namely glfwSwapBuffers() takes a huge amount of time to render every X cycles. For example it would render 8 frames in 0.0003 seconds each, and then glfwSwapBuffers would take 0.4 seconds on the 9th iteration.

One of the suspects was double buffering, so I wanted to disable it. Alas, when I try to - it throws in an error "EGL: Failed to find a suitable EGLConfig"

Upon digging through your code, I found the likely suspect in egl_context.c chooseEGLConfig, line 176
Basically, as far as I understand, you retrieve the available buffers, but then instead of reading the doublebuffer value from EGL, like with all other parameters, you just go **u->doublebuffer = GLFW_TRUE;**

maybe I'm wrong and it's not the place, but you should look at it.

**Comment by elmindreda** [comment]
Thank you for reporting this! It should be fixed now with 114776a24605418e6d719d2f30141e351e93c6e0, which should be included in the next patch release.

**Comment by ItIsApachee** [comment]
I found myself dealing with the same issue of not being able to use single buffering with EGL. I'm using ANGLE as EGL, and GLES implementation. After disabling double buffering, I recieve an error "EGL: Failed to create window surface: Arguments are inconsistent".
